packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -69,7 +69,7 @@ const BuySellHeader = ({ is_visible, table_type, setTableType }) => {
             </div>
             <div className='buy-sell__header-match-ads'>
                 <Checkbox
-                    label={localize('Show the matching ads')}
+                    label={<Localize i18n_default_text='Show the matching ads' />}
```

Comment:
Why? these are the same. For translation cards we either need to merge the latest translation PR or check with translator if the texts has been changes.